frontend-react/src/components/Admin/AdminReceiverDashboard.test.tsx

```diff
@@ -1,11 +1,15 @@
 import { fireEvent, screen } from "@testing-library/react";
 import { NetworkErrorBoundary } from "rest-hooks";
 import React, { Suspense } from "react";
-import { MemoryRouter } from "react-router-dom";
+import { Fixture } from "@rest-hooks/test";
 
-import { AdmConnStatusDataType } from "../../resources/AdmConnStatusResource";
+import {
+    AdmConnStatusDataType,
+    AdmConnStatusResource,
+} from "../../resources/AdmConnStatusResource";
 import { ErrorPage } from "../../pages/error/ErrorPage";
-import { renderWithBase } from "../../utils/CustomRenderUtils";
+import { renderApp } from "../../utils/CustomRenderUtils";
+import Spinner from "../Spinner";
 
 import { _exportForTesting } from "./AdminReceiverDashboard";
```

Comment:
Minor point and COMPLETELY out of the scope of this pull request because it already existed beforehand, but I don't think we should use this pattern going forward; if things need to be exported, they should just be exported 🤷 We should also break large files like this up into smaller files in the future.